awx/main/models/events.py

```diff
@@ -368,10 +380,11 @@ def _send_notifications():
             value = force_text(event_data.get(field, '')).strip()
             if value != getattr(self, field):
                 setattr(self, field, value)
-        analytics_logger.info(
-            'Event data saved.',
-            extra=dict(python_objects=dict(job_event=self))
-        )
+        if settings.LOG_AGGREGATOR_ENABLED:
```

Comment:
Our custom filter technically checks this, but it does so *after* making the logging record and looping through all the handlers (which is actually pretty expensive).